lib/utils.js

```diff
@@ -593,7 +597,39 @@ const _licenseUrlOverrides = [
   { test: /^https?:\/\/specflow.org\/plus\/eula\//i, license: 'OTHER' },
   { test: /^https?:\/\/www.streamcoders.com\/products\/msneteula.html/i, license: 'OTHER' },
   { test: /^https?:\/\/workflowenginenet.com\/EULA/i, license: 'OTHER' },
-  { test: /^https?:\/\/aka.ms\/azureml-sdk-license/i, license: 'OTHER' }
+  { test: /^https?:\/\/aka.ms\/azureml-sdk-license/i, license: 'OTHER' },
+  {
+    test: /^\w*https?:\/\/(?:www\.)?eclipse\.org\/legal\/epl-v10\.html$/i,
+    license: 'EPL-1.0'
+  },
+  {
+    test: /^https?:\/\/(?:www\.)?eclipse\.org\/legal\/epl-2\.0\/?$/i,
+    license: 'EPL-2.0'
+  },
+  {
+    test: /^\w*https?:\/\/(?:www\.)?mozilla\.org(?:\/en-US)?\/MPL\/(MPL-1\.1\.html|1\.1\/)$/i,
+    license: 'MPL-1.1'
+  },
+  {
+    test: /^https?:\/\/(?:www\.)?mozilla\.org(?:\/en-US)?\/MPL\/2\.0\/$/i,
+    license: 'MPL-2.0'
+  },
+  {
+    test: /^\w*https?:\/\/(?:www\.)?eclipse\.org\/org\/documents\/edl-v10\.php$/i,
+    license: 'BSD-3-Clause'
+  },
+  {
+    test: /^https?:\/\/glassfish\.(dev\.java\.net|java\.net)\/public\/CDDL\+GPL_1_1\.html$/i,
+    license: 'CDDL-1.1 or GPL-2.0-only WITH Classpath-exception-2.0'
```

Comment:
the or operator should be capitalized, e.g. `OR`.  See [specification](https://spdx.github.io/spdx-spec/v3.0.1/annexes/spdx-license-expressions/#case-sensitivity)

Reply:
Done.